Tests/VectorUtils.h

```diff
@@ -24,4 +24,21 @@ void fillVectorRandomly(Vector& x)
     for (Index i = 0; i < size; ++i) {
         MathLib::setVector(x, i, rnd(random_number_generator));
     }
+#ifdef USE_PETSC
+    x.finalizeAssembly();
+#endif
+}
+
+template <typename Vector>
```

Comment:
For a global vector x filled with a constant a, you can use 
```
 MathLib::LinAlg::set(x, a);
```

Besides, this function is not used in this PR.